src/libs/Logger.js

```diff
@@ -0,0 +1,65 @@
+const pino = require('pino');
+const os = require('os');
+const loadJson = require('../helpers/loadJson');
+
+const packageInfo = loadJson('package.json');
+const versionInfo = loadJson('version.json', {});
+
+const levels = ['debug', 'info', 'warn', 'error', 'fatal'];
+
+module.exports = class Logger {
+  constructor ({
+    logLevel = '',
+    destination = pino.destination(1),
+    pretty = false,
+    stackLevel = 'error'
+  } = {}) {
+    const { projectName, buildNumber, commit } = versionInfo;
+
+    this.main = pino({
+      name: packageInfo.name,
+      base: {
+        hostname: os.hostname(),
+        pid: process.pid,
+        version: packageInfo.version,
+        projectName,
+        buildNumber,
+        commit
+      },
+      level: logLevel || 'info',
+      timestamp: () => `,"time":"${new Date().toISOString()}"`,
+      serializers: {
+        error: pino.stdSerializers.err
+      },
+      prettyPrint: pretty ? { colorize: true } : false
+    }, destination);
+
+    this.stackLevelIndex = levels.indexOf(stackLevel);
+  }
+
+  _buildLog (logLevel, event, data = null, meta = null) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this '_buildLog' function to match the regular expression ^[a-z][a-zA-Z0-9]*$ [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=javascript%3AS100)